client/test/test_archive.py

```diff
@@ -29,6 +29,21 @@ def test_push_file(archive, random_metadata, tmpfile, s3_key):
     assert common_keys == set(random_metadata.keys())
 
 
+def test_push_large_file(
+        monkeypatch, archive, random_metadata, tmpfile, s3_key):
+    monkeypatch.setenv('DATALAKE_CHUNK_SIZE_MB', 1)
+    expected_content = ('mwahaha' * 1024 * 1024).encode('utf-8')
```

Comment:
Can we randomize the input more and ensure that it's not aligned to a chunk boundary? Chunk-aligned data can obfuscate bugs.